awsdynamodb_describe_backup.html

```diff
@@ -0,0 +1,285 @@
+        text-align: center;
+        border: 1px solid;
+        text-transform: uppercase;
+        font-weight: normal;
+        padding: 5px;
+    }
+
+    .awsdynamodb .wf-table-horizontal td {
+        border: 1px solid;
+        padding: 5px;
+        padding-left: 4px;
+    }
+
+    .awsdynamodb .wf-h3-style {
+        font-size: 20px
+    }
+
+    .awsdynamodb .wf-h4-style {
+        font-size: 16px
+    }
+
+    .awsdynamodb .collapse.in {
+        display: block !important;
+    }
+
+    .awsdynamodb .panel-collapse {
+        overflow-x: auto;
+    }
+
+    .awsdynamodb .glyphicon.glyphicon-dot:before {
+        content: "\25cf";
+        font-size: 10px;
+    }
+
+    .awsdynamodb a.nowrap {
+        white-space: nowrap;
+    }
+
+    tbody {
+        counter-reset: serial-number;
+        /* Set the serial number counter to 0 */
+    }
+
+    #tb2:before {
+        counter-increment: serial-number;
+        /* Increment the serial number counter */
+        content: counter(serial-number);
+        /* Display the counter */
+    }
+
+    .boldtext {
+        font-weight: bold;
+    }
+</style>
+<div class="awsdynamodb" style="overflow: auto; width: 100%; height: 100%; padding-left:10px; padding-right:10px">
+    <!-- Main Div -->
+    {% for result in results %}
+    <!-- loop for each result -->
+    <br>
+
+    <!------------------- For each Result ---------------------->
+    {% if not result.data %}
+    <h4 class="wf-h4-style">No data found</h4>
+    {% else %}
+
+    <h3 class="wf-h3-style">Describe Backup</h3>
+    <div class="metadata_div">
+        <!--Default View-->
+        <div class="panel-group" id="server-information">
+            <div class="panel">
+                <div id="default-view-table" class="panel-collapse collapse in">
+                    <table class="wf-table-horizontal datatable">
+                        <thead>
+                            Backup Table Details
+                        </thead>
+                        <tbody>
+                            <tr>
+                                <td>Backup Details</td>
+                                <td>
+                                    <table class="wf-table-horizontal datatable">
+                                        <tr>
+                                            <td class="boldtext">Backup Name</td>
+                                            <td>{{result.data.0.BackupDescription.BackupDetails.BackupName}}</td>
+                                        </tr>
+                                        <tr>
+                                            <td class="boldtext">Backup ARN</td>
+                                            <td>
+                                                <a href="javascript:;"
+                                                    onclick="context_menu(this, [{'contains': ['aws dynamodb backup arn'], 'value': '{{result.data.0.BackupDescription.BackupDetails.BackupArn}}' }], 0, {{ container.id }}, null, false);">
```

Comment:
Are we getting any value in the custom view? There is no body text for the anchor tag.

Reply:
Have added the value to be displayed